4.0.0.4/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM buildpack-deps:trusty
-MAINTAINER Henrik Feldt
+LABEL maintainer "Dave Curylo <curylod@asme.org>, Steve Desmond @stevedesmond_ca"
```

Comment:
feel free to use steve@stevedesmond.ca in lieu of my Twitter handle, though not sure it matters since it's just a `LABEL` now -- or I'm fine with keeping it `MAINTAINER` with just one or the other